### ISSUE TITLE
Cocoapods Spec Uses the Wrong Repo For Pluralize

### ISSUE THREAD

**Issue by hirad** [joshualat/Pluralize.swift#14]
While the podspec on this repo references joshualat/Pluralize.swift, the one that you get by doing `pod 'Pluralize.swift'` is actually @sammy-SC's fork, which hasn't been updated for a while and is still using Swift 1.2. 

I'm not quite sure what the process is for fixing that at the moment, just figured I'd log it here.

**Comment by jordanmaguire** [comment]
Yes, wonderful. For others coming across this you can specify the repo in your Podfile like this:

``` ruby
pod 'Pluralize.swift',
  git: 'git@github.com:joshualat/Pluralize.swift.git',
  branch: 'master'
```

**Comment by paulnicholson** [comment]
@joshualat @sammy-SC Can you push an update to cocoapods?

**Comment by DavidLari** [comment]
@paulnicholson This repo appears to be abandoned. Best bet is to clone it and maintain yourself or simply grab the Pluralize.swift source file and drop it into your own project.

**Comment by paulnicholson** [comment]
@DavidLari Using this repo in the podfile like ```pod 'Pluralize.swift', git: 'git@github.com:joshualat/Pluralize.swift.git', branch: 'master'``` works just fine. The podspec in the official cocoapods repo points to @sammy-SC's fork which is outdated.

**Comment by DavidLari** [comment]
That's true, but the 'maintainer' has shown no interest in responding to any issues, thus I would consider it abandoned and plan accordingly. In my case, I just copied the relevant source file into my project and edit as needed.

**Comment by sammy-SC** [comment]
Hello @DavidLari @paulnicholson ,

I apologize for late response.

I believe the project has been abandoned and I recommend the same thing as @DavidLari
